pandas/core/generic.py

```diff
@@ -7563,7 +7584,8 @@ def resample(self, rule, how=None, axis=0, fill_method=None, closed=None,
         For DataFrame objects, the keyword ``on`` can be used to specify the
         column instead of the index for resampling.
 
-        >>> df = pd.DataFrame(data=9*[range(4)], columns=['a', 'b', 'c', 'd'])
+        >>> df = pd.DataFrame(data=9 * [range(4)],
+        ...                   columns=['a', 'b', 'c', 'd'])
         >>> df['time'] = pd.date_range('1/1/2000', periods=9, freq='T')
         >>> df.resample('3T', on='time').sum()
                              a  b  c  d
```

Comment:
We should show the data before resampling, otherwise it's difficult to see what `resample` is doing. Also, we can probably have just two columns instead of 4, name them with something meaningful (like `price` and `volume` instead of `a` and `b`), and create the `DataFrame` in a nicer way (in a single command, with no `range`, may be just a dict).

Reply:
OK, I've revamped this example. Pushing now altogether with the rest of the changes. Agree more meaningful examples make sure everyone understands. Just a bit conscious of being too verbose. But looking forward to your feedback.